sdk/keyvault/keyvault-keys/swagger/README.md

```diff
@@ -6,7 +6,7 @@
 typescript:
   package-name: "@azure/keyvault-keys"
 use-extension:
-  "@microsoft.azure/autorest.typescript": "~5.0.1"
+  "@autorest/typescript": "6.0.0-dev.20200623.2"
```

Comment:
I took this version from the previous PR I did a while ago. Is this the Good Version ™? cc: @joheredi

Reply:
Can you try with  `6.0.0-dev.20200826.1` please? this is the latest one